cmd/apigen/go.mod

```diff
@@ -8,14 +8,15 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/kcp-dev/kcp/pkg/apis v0.10.1-0.20230123155139-7b3126cbf96c
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/pflag v1.0.5
+	github.com/spf13/pflag v1.0.6-0.20210604193023-d5e0c0615ace
```

Comment:
What's causing this bump, as well as the other bumps in this file?

Reply:
go mod tidy to force update sum file ;/ and same for bellow. 
how worries you are for these. I can try remove them, but it playing with `go get && go mod tidy` x repeat until you match version :/

Reply:
I tried to reduce the scope for more core packages, but these still gets bumped by tooling

Reply:
I was thinking that if the versions matched across the go.mod files, we wouldn't see the bumps. I can try to align them if you'd like.

Reply:
let me try once more tomorrow :)

Reply:
Ah, I think this is what you said, root go mod has this version, to it was aligned as you stated. https://github.com/kcp-dev/kcp/blob/main/go.mod#L30